src/components/button/button.ts

```diff
@@ -123,6 +143,7 @@ export class Button extends LitElement {
   _btnEl!: any;
 
   override render() {
+    this._reSizeButton();
```

Comment:
It looks like this is executing on every render. A better place to put it would be within the debounceResize listener.